src/buffer/hexutil.cpp

```diff
@@ -44,25 +45,20 @@ namespace scale {
     return str;
   }
 
-  std::string hex_upper(const gsl::span<const uint8_t> bytes) noexcept {
+  std::string hex_upper(const RangeOfBytes auto &bytes) noexcept {
```

Comment:
Revert `concept` to `span`, or move to .cpp (or remove unused),
otherwise .cpp does not instantiate any implementation (link error)